inngest-core/src/main/kotlin/com/inngest/Comm.kt

```diff
@@ -108,20 +109,29 @@ class CommHandler(
     }
 
     fun register(): String {
-        // TODO - This should detect the dev server or use base url
-        val registrationUrl = "http://localhost:8288/fn/register"
+        val registrationUrl = "${config.baseUrl()}/fn/register"
+        // TODO use name from ServeConfig, framework from adapter, serveOrigin from ServeConfig
         val requestPayload =
             RegistrationRequestPayload(
                 appName = "my-app",
                 framework = "ktor",
-                sdk = "kotlin",
+                sdk = "inngest-kt",
```

Comment:
kotlin here is fine. we generally know which SDK it is.

Reply:
I was making it consistent with https://github.com/inngest/inngest-kt/blob/741ca45eb5b81ef9dd8195a698007f079feef4b2/inngest-core/src/main/kotlin/com/inngest/Environment.kt#L5 and https://github.com/inngest/inngest/blob/main/docs/SDK_SPEC.md#411-definitions, although I know technically this isn't the same as the `X-Inngest-Sdk` header

Reply:
yeah, that's actually `$lang:$sdk_version` so it shouldn't be called `inngest-kt` because no other SDKs calls it this way right now.
the spec is not valid in this case, so I'll bring it up.

Reply:
you know what, let's keep it as `inngest-kt` for now, since I did see it in other SDKs, though I need to verify some stuff.